packages/radix-vue/src/TagsInput/TagsInputRoot.vue

```diff
@@ -172,6 +172,10 @@ provideTagsInputRootContext({
       case 'End':
       case 'ArrowRight':
       case 'ArrowLeft': {
+        // allow text selection while maintaining `shift` key
+        if(event.shiftKey)
+          break
+
```

Comment:
I think we don't need it for `Tags Input`. The current implementation works well, where we only focus on `TagsInputItem` when we are not highlighting any text

https://github.com/user-attachments/assets/da69a838-8aaa-4df6-936f-4f652e7a3c97

Reply:
I believe you used arrows to select char one by one. This one enables text selection in its entirety or what's left of it with shift+home or shift+end. That is the expected behaviour when using a text input.

Reply:
[Screencast from 2024-10-02 20-53-08.webm](https://github.com/user-attachments/assets/61e45082-5918-45c6-b261-1f9a6435edbb)

Reply:
What about the following behavior ? I removed the tests I wrote because I think they're not great and couldn't figure it out.

[Screencast from 2024-10-02 21-52-00.webm](https://github.com/user-attachments/assets/82e9a914-8d64-4533-8664-011ea0b15dca)

[Screencast from 2024-10-02 21-55-07.webm](https://github.com/user-attachments/assets/528bdaf0-b37f-4c6d-b3c5-5523d53a3148)